src/Model/ValueObject/Calendar/SubEventUpdate.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Model\ValueObject\Calendar;
+
+final class SubEventUpdate
```

Comment:
The class `SubEventUpdate` inside the `CultuurNet\UDB3\Event\ValueObjects` namespace is rather new. Maybe we can discard `CultuurNet\UDB3\Event\ValueObjects\SubEventUpdate` and directly start using `CultuurNet\UDB3\Model\ValueObject\Calendar\SubEventUpdate`? I believe we wanted to make more use of the `Udb3Models` and avoid duplicating VOs?

Reply:
Done!